src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2688,6 +2688,57 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/datasets/{id}/createSnapshotRequest:
+    post:
+      tags:
+        - datasets
+      description: Create a new snapshot request.
```

Comment:
I wonder if a user journey type description would be helpful here. Something like "As a researcher, create a snapshot request for a dataset, given a set of criteria and required concept sets and values."?